pkgs/data/fonts/helvetica-neue-lt-std/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, unzip }:
+
+stdenv.mkDerivation {
+  name = "helvetica-neue-lt-std";
+
+  src = fetchurl {
+    url = "http://www.ephifonts.com/downloads/helvetica-neue-lt-std.zip";
+    sha256 = "0nrjdj2a11dr6d3aihvjxzrkdi0wq6f2bvaiimi5iwmpyz80n0h6";
+  };
+
+  nativeBuildInputs = [ unzip ];
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  sourceRoot = "Helvetica Neue LT Std";
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/opentype
+    cp -v *.otf $out/share/fonts/opentype
+  '';
+
+  meta = {
+    description = "Helvetica Neue LT Std font";
+    homepage = http://www.ephifonts.com/free-helvetica-font-helvetica-neue-lt-std.html;
+    license = stdenv.lib.licenses.free;
```

Comment:
I don't believe this package is free in the way `stdenv.lib.licenses.free` states. Running `otfinfo` on one of the files in the archive I get
```
rycee@beta:~…std/Helvetica Neue LT Std$ otfinfo -i HelveticaNeueLTStd-BdCnO.otf
Family:              HelveticaNeueLT Std Cn
Subfamily:           Bold Italic
Full name:           HelveticaNeueLTStd-BdCnO
PostScript name:     HelveticaNeueLTStd-BdCnO
Preferred family:    Helvetica Neue LT Std
Preferred subfamily: 77 Bold Condensed Oblique
Mac font menu name:  HelveticaNeueLT Std Bold Cn Ob
Version:             OTF 1.029;PS 001.000;Core 1.0.33;makeotf.lib1.4.1585
Unique ID:           1.029;ADBE;HelveticaNeueLTStd-BdCnO
Designer:            Linotype Staff
Vendor URL:          http://www.adobe.com/type
Trademark:           Helvetica is a trademark of Heidelberger Druckmaschinen AG, exclusively licensed through Linotype Library GmbH, and may be registered in certain jurisdictions.
Copyright:           Copyright © 1990, 2002 Adobe Systems Incorporated.  All Rights Reserved. © 1981, 2002 Heidelberger Druckmaschinen AG. All rights reserved.
License URL:         http://www.adobe.com/type/legal.html
Vendor ID:           ADBE
```

The "All Rights Reserved" is a bit concerning and I think `licenses.unfree` probably is a safer choice.